### PR TITLE
feat: add avatar selection modal to sign/login flow

### PR DESCRIPTION
**Pull request by nickreynolds**
- also display user avatar in navbar if available
- selection dialog not to-design, just placeholder

### CODE REVIEW

File: packages/components/src/Account/Auth/UserSetAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        ],
+      });
+
+      if (res.data && res.data.channelsSetAvatar && res.data.channelsSetAvatar.id === channelID) {
+        if (this.props.onSetAvatarComplete) {
+          this.props.onSetAvatarComplete();
+        }
+      }
+      if (res.error) {
+        console.log("res.error: ", res.error);
```
Comment (tobek) [inline packages/components/src/Account/Auth/UserSetAvatar.tsx]: Minor, but should this set `errorMessage` state too?

Comment (nickreynolds) [reply packages/components/src/Account/Auth/UserSetAvatar.tsx]: I'll address this when we build the *real* avatar selector

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.20 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**